Makefile.am

```diff
@@ -16,7 +16,7 @@ CODE_COVERAGE_IGNORE_PATTERN = "*-generated.c"
 GC_CFLAGS = -fdata-sections -ffunction-sections
 GC_LDFLAGS = -Wl,--gc-sections -Wl,-Map,$@.map
 AM_CFLAGS = -DG_LOG_DOMAIN=\"rauc\" $(GC_CFLAGS) $(WARN_CFLAGS) $(GLIB_CFLAGS) $(CURL_CFLAGS) $(NL3_CFLAGS) $(OPENSSL_CFLAGS)
-AM_LDFLAGS = $(GC_LDFLAGS) $(WARN_LDFLAGS) $(GLIB_LDFLAGS) $(CURL_LDFLAGS) $(OPENSSL_LDFLAGS)
+AM_LDFLAGS = $(GC_LDFLAGS) $(WARN_LDFLAGS)
```

Comment:
Implementation looks good. You might want to mention that PKG_CHECK_MODULES adds everything to _LIBS. Without seeing the code, the comment about librauc_la_LIBADD is a bit confusing.

Reply:
Thanks.